### ISSUE TITLE
Text resizes causes loss of content for Quickview at smaller viewports

### ISSUE THREAD

**Issue by ravenokko** [shopware/platform#16030]
This is a violation of [WCAG S.C. 1.4.4](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html)

**To reproduce:**

- In Firefox go to preferences and search for Zoom
  - tick Zoom text only
- set viewport to 320px wide
- Zoom to 200%

**Solution should resolve the following**

- [ ] Quickview overlay should be centered
- [ ] `Show more details` button text should wrap
- [ ] Ensure the arrows either do not permanently block any text, or text wraps accordingly

<img src="https://github.com/user-attachments/assets/167c4ac0-259f-43b1-a702-8fda8e823165" width=30% height=30% />
<img src="https://github.com/user-attachments/assets/60ff634d-a3de-4250-8351-ef87f1132fdf" width=30% height=30% />
